pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -912,6 +912,28 @@ func Convert_v1_Usbredir_To_api_Usbredir(vmi *v1.VirtualMachineInstance, domainD
 	return true, nil
 }
 
+func Convert_v1_Sound_To_api_Sound(vmi *v1.VirtualMachineInstance, domainDevices *api.Devices, _ *ConverterContext) {
+	sound := vmi.Spec.Domain.Devices.Sound
+
+	// Default is to not have any Sound device
+	if sound == nil {
+		return
+	}
+
+	model := "ich9"
+	if strings.ToLower(sound.Model) == "ac97" {
```

Comment:
Why do we need `ToLower` here?

Reply:
Not a must. I think I was trying to avoid error with AC97 too. I'll remove it.